packages/developer-portal/src/components/pages/analytics-v2/installations/installations-by-app-chart.tsx

```diff
@@ -0,0 +1,117 @@
+import 'chart.js/auto'
+import { AppSummaryModelPagedResult, InstallationModelPagedResult } from '@reapit/foundations-ts-definitions'
+import React, { FC, useMemo } from 'react'
+import { Chart } from 'react-chartjs-2'
+import { useAnalyticsState } from '../state/use-analytics-state'
+
+export interface InstallationsByAppChartProps {
+  installations: InstallationModelPagedResult
+}
+
+export interface ChartDataModel {
+  labels: string[]
+  data: number[]
+}
+
+export const handleSortChartData =
+  (installations: InstallationModelPagedResult, apps: AppSummaryModelPagedResult | null) => () => {
+    if (!installations.data || !apps?.data) {
+      return {
+        labels: [],
+        data: [],
+      }
+    }
+
+    const sortedInstallsByApp = installations.data.reduce<{ [key: string]: number }>((current, { appId }) => {
+      const name = apps.data?.find((app) => app && app.id === appId)?.name ?? ''
+
+      if (current[name]) {
+        current[name] += 1
+        return current
+      }
+
+      current[name] = 1
```

Comment:
just avoids extra return in condition

```suggestion
     current[name] = (current[name] || 0) + 1
```

Reply:
Agreed on this, will update both reduce functions 👍🏻